unified_targets/configs/MATEKF722SE.config

```diff
@@ -0,0 +1,220 @@
+# dump hardware
+
+# version
+# Betaflight / MATEKF722SE (MF7S) 4.0.0 Mar  9 2019 / 10:27:45 (49390594d) MSP API: 1.41
+
+board_name MATEKF722SE
+manufacturer_id MTKS
+
+# resources
+resource BEEPER 1 C13
+resource MOTOR 1 B04
+resource MOTOR 2 B05
+resource MOTOR 3 B00
+resource MOTOR 4 B01
+resource MOTOR 5 A15
+resource MOTOR 6 B03
+resource MOTOR 7 B06
+resource MOTOR 8 B07
+resource SERVO 1 NONE

```

Comment:
All of these `NONE` lines should be removed. Default settings in the `# master` should be removed as well.

Easiest to load this dump into a board running the STM32F7X2 Unified Target, and then do a `diff all bare`, according to https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/CreatingAUnifiedTarget.md#6-create-a-unified-target-configuration-file-for-your-board

Reply:
True, the problem when pasting the baro lines distracted me from the rest of the guide. Fixed.